ReminderApp-OBJC/ReminderApp-OBJC/Classes/Controller/CDReminderTableViewController.m

```diff
@@ -11,12 +11,14 @@
 #import "CDReminderCell.h"
```

Comment:
Observed during testing. The add reminder cell shouldn't have the option to swipe and delete.